usr/lib/common/key.c

```diff
@@ -6194,6 +6194,80 @@ CK_RV des3_wrap_get_data(TEMPLATE *tmpl,
     return CKR_OK;
 }
 
+//  aes_xts_set_default_attributes()
+//
+CK_RV aes_xts_set_default_attributes(TEMPLATE *tmpl, TEMPLATE *basetmpl, CK_ULONG mode)
```

Comment:
this seems to be a copy of function aes_set_default_attributes more or less, except a very few places.
I don't really like tat much code duplication.
I would suggest to modify aes_set_default_attributes and add a key type parameter to it.
All its callers then simply pass the desired key type to it.